compile.js

```diff
@@ -415,3 +462,72 @@ function getDefaultWriteTest(ctx, field) {
 function isPacked(field) {
     return field.options.packed === 'true';
 }
+
+function getTsType(field) {
+    let type = field.type;
+
+    if (fieldShouldUseStringAsNumber(field)) type = 'string';
+    else if (fieldTypeIsNumber(field)) type = 'number';
+    else if (field.type === 'bytes') type = 'Uint8Array';
+    else if (field.type === 'bool') type = 'boolean';
+
+    return field.repeated ? `Array<${type}>` : type;
```

Comment:
Nit: the shorter syntax is more common I guess?
```suggestion
    return field.repeated ? `${type}[]` : type;
```

Reply:
This is more a matter of taste, but shorter syntax looks good here 👍